integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'source': u'openstreetmap.org',
+                'surface': u'paved',
+            }),
+            dsl.relation(1, {
+                'distance': u'135.38',
+                'name': u'Jalan Batu Pahat - Ayer Hitam - Jemaluang',
+                'network': u'my:federal',
+                'ref': u'50',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikipedia': u'en:Malaysia Federal Route 50',
+            }, ways=[553323706]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 553323706,
+                'network': u'MY:federal',
+            })
+
+    def test_nofylkesvei(self):
+        import dsl
+
+        z, x, y = (16, 33706, 18951)
+
+        self.generate_fixtures(
+            dsl.is_in('NO', z, x, y),
+            # https://www.openstreetmap.org/way/106337203
+            dsl.way(106337203, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'maxspeed': u'50',
+                'ref': u'555',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'name': u'Fylkesvei 555 (Hordaland)',
+                'network': u'no:Fylkesvei',
+                'operator': u'Hordaland Fylkeskommune',
+                'ref': u'555',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q6367615',
+                'wikipedia': u'no:Fylkesvei 555',
+            }, ways=[106337203]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 106337203,
+                'network': u'NO:Fylkesvei',
+            })
+
+    def test_noriksvei(self):
+        import dsl
+
+        z, x, y = (16, 34099, 18283)
+
+        self.generate_fixtures(
+            dsl.is_in('NO', z, x, y),
+            # https://www.openstreetmap.org/way/521803129
+            dsl.way(521803129, dsl.tile_diagonal(z, x, y), {
+                'bridge': u'yes',
+                'highway': u'trunk',
+                'layer': u'1',
+                'maxspeed': u'70',
+                'ref': u'15',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'name': u'Riksvei 15',
+                'network': u'no:riksvei',
+                'operator': u'Statens vegvesen',
+                'ref': u'Rv15',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q6396950',
+                'wikipedia': u'no:Riksvei 15',
+            }, ways=[521803129]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 521803129,
+                'network': u'NO:Riksvei',
```

Comment:
Does the shield text need a transform and test?

Reply:
(The next test example seems similar but just has ref value of 7 excluding the Rv prefix. Let's be consistent.)

Reply:
Added tests in 436d1bb. Worth pointing out there's more extensive tests of Norway's roads in [`integration-test/1491-norwegian-shields.py`](https://github.com/tilezen/vector-datasource/pull/1515/files/28c06004411a57c4fabc29241d7315d81e182fc6#diff-2dcb4bdaeb11a50e6c101facf8f9ad66).